crates/fuel-core/src/service/config.rs

```diff
@@ -79,8 +84,8 @@ impl Config {
             database_type: DbType::RocksDb,
             #[cfg(not(feature = "rocksdb"))]
             database_type: DbType::InMemory,
+            debug: true,
```

Comment:
Are we sure that we want to always have debug enabled by default? Could this lead to unexpected results in some tests?

Reply:
It only broke the test that expected `manual_blocks_enabled = false`. It is `true` by default to not change the behavior for `utxo_validation`